kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -1,4 +1,3 @@
-/*global define*/
 /*jslint white:true,browser:true*/
```

Comment:
we don't need this jslint declaration either -- although I'll be removing it in job status table part MXCVII, so it doesn't matter too much